src/daft-local-execution/src/channel.rs

```diff
@@ -16,6 +16,10 @@ impl<T> Receiver<T> {
     pub(crate) fn blocking_recv(&self) -> Option<T> {
         self.0.recv().ok()
     }
+
+    pub(crate) fn into_inner(self) -> loole::Receiver<T> {
```

Comment:
question: does loole have great perf characterisics? why are we using it?

Reply:
Performance wise it's decently better, especially for single capacity channels (which is what most of the executor channels are). It's also a MPMC channel, which tokio doesn't provide.